pr_check.sh

```diff
@@ -15,21 +17,21 @@ COMMON_BUILDER=https://raw.githubusercontent.com/RedHatInsights/insights-fronten
 
 set -ex
 
-docker run -t \
-  -v $PWD:/e2e:ro,Z \
-  -w /e2e \
-  -e CHROME_ACCOUNT=$CHROME_ACCOUNT \
-  -e CHROME_PASSWORD=$CHROME_PASSWORD \
-  --add-host stage.foo.redhat.com:127.0.0.1 \
-  --add-host prod.foo.redhat.com:127.0.0.1 \
-  --entrypoint bash \
-  quay.io/cloudservices/cypress-e2e-image:06b70f3 /e2e/run-e2e.sh
+#docker run -t \
```

Comment:
This is temporarily commented-out for the sake of speeding up trial/error with pr_check script changes